apps/eclipse/eclipse_win.py

```diff
@@ -42,9 +42,16 @@ def toggle_comment():
 
 @ctx.action_class("edit")
 class EditActions:
-    def find(text: str):
+    def find(text: str = None):
         actions.key("ctrl-f")
-        actions.insert(text)
+        if text:
+            actions.insert(text)
```

Comment:
I believe this matches the default Windows implementation, right? Should we just remove the matching ones?

Reply:
Yes we should probably have an operating system defaults not every single application needs to specify this. I'm on it.

Reply:
Done